index.bs

```diff
@@ -298,17 +301,14 @@ type: dfn
   </section>
 
   <section>
-    <h3 id="existing-mix-algorithms">Existing Mixed Content Algorithms and Modifications</h3>
+    <h3 id="existing-mix-algorithms">Modifications to previous algorithms</h3>
```

Comment:
No need to refactor this in this PR, but since there's no longer a "previous" spec to modify, I think it'll make sense to just incorporate these algorithms into the primary Algorithms section instead of a subsection.

Reply:
> No need to refactor this in this PR, but since there's no longer a "previous" spec to modify, I think it'll make sense to just incorporate these algorithms into the primary Algorithms section instead of a subsection.

I would think so too — but yeah that’s out of scope for this PR at least. I leave it up to @carlosjoan91 to make a call whether and when to actually do that

Reply:
Yeah, incorporating these into algorithms sgtm. I can put a PR together once this one is merged.